netmap/policy.go

```diff
@@ -30,6 +30,46 @@ type PlacementPolicy struct {
 	replicas []netmap.Replica
 }
 
+func copyFilter(f netmap.Filter) netmap.Filter {
+	var filter netmap.Filter
+
+	filter.SetName(f.GetName())
+	filter.SetKey(f.GetKey())
+	filter.SetOp(f.GetOp())
+	filter.SetValue(f.GetValue())
+
+	filters := make([]netmap.Filter, len(f.GetFilters()))
```

Comment:
`nil` becomes an empty slice, are we ok with this? e.g. what `require.Equal` will return?